### DISCUSSION TITLE
Why <div /> is a thing? and can we stop this from happneing.

### DISCUSSION THREAD

**Discussion by pmnzt** [antfu/eslint-config#567] [Q&A]
Every time i add a div ``<div></div>``, if i don't put anything inside that div the auto linter will replace that with ``<div />``, which makes no sense and creates issues, as we all know the ``<div />`` is not a thing and causes issues as the browser considers this just exactly as only having ``<div>``.

Why this behavior is included ? and is there any way to fix it ?

**Comment by Dimava** [comment]
1. Comment out eslint.rules.customizations
2. Find what rule fixes it
3. Change that rule settings or disable it altogether

Vue components often do not have content but have long names so self-closing is preferred for them
